docs/source/features/parallelisms.rst

```diff
@@ -112,6 +130,16 @@ NeMo integrates Tensor Parallelism through the implementation from Megatron Core
 
 For detailed API usage and additional configurations, consult the `Megatron Core Developer Guide <https://docs.nvidia.com/Megatron-Core/developer-guide/latest/api-guide/tensor_parallel.html>`_.
 
+FSDP + Tensor Parallelism
+~~~~~~~~~
+
+NeMo supports FSDP along with tensor parallelism. This is done by restricting the model state sharding to the data-parallel domain.
+Using FSDP with tensor parallelism can be helpful when the model doesn't have sufficient parallelism to deploy on a large scale training system with the data-parallel mapping; e.g., running a model with the global batch size of 1024 on 2048 GPUs.
```

Comment:
In NeMo, FSDP can be combined with Tensor Parallelism. This involves sharding the model state specifically within the data-parallel domain. Using FSDP alongside TP is useful when the model lacks sufficient parallelism to be deployed on a large-scale training system using data-parallel mapping. For instance, it allows running a model with a global batch size of 1024 across 2048 GPUs.